### DISCUSSION TITLE
Open Shell Start menu with same look and feel for all users

### DISCUSSION THREAD

**Discussion by MPJ77** [Open-Shell/Open-Shell-Menu#2038] [Q&A]
i have read and tried to follow the instructions on how to set up a start menu using Open Shell on Windows 11 so that it looks the same for all users, but I have not been able to make it work.
I have tried using the registry and adding changes to HKEY_LOCAL_MACHINE as well as HKEY_CURRENT_USER and the menus were never the same.
I tried using start2.bin file and that did not help either.
I am trying to create a simple start menu that will be the same for all users.
I am trying to do this using the Windows 7 style two column menu.
All I want is a start menu that has no icon shortcuts on the menu left side.
When you click on the start button all I want on the left side is all programs.
On the right-hand side, I want to remove Documents, pictures, Music, Recent Items, Settings, Default Programs and Help and Support.
I can accomplish this manually by going into the settings and making these changes.
The problem is I am not able to figure out how to apply these changes to all users.
Can anyone provide some assistance to help me achieve my goal?

Thanks!

**Comment by ge0rdi** [comment]
I'd suggest to have a look at `Administrative Settings` section in `Open-Shell`'s help.

**Comment by MPJ77** [comment]
I looked at the Administrative Settings section in Open-Shell's help, but it does not provide the information that I need.

**Reply by ge0rdi** [reply]
So can you provide exact example of what doesn't work for you?
Like examples of values you create in HKLM key and that are not applied for the users?
Exact registry path, names, values. So that we can try to replicate.

**Comment by MPJ77** [comment]
These are the registry values I am using.
One is for HKEY_CURRENT_USERS and the other one I added to HKEY_LOCAL_MACHINE.
 
 
**### HKCU**
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\OpenShell]

[HKEY_CURRENT_USER\Software\OpenShell\ClassicExplorer]
"ShowedToolbar"=dword:00000001
"NewLine"=dword:00000000

[HKEY_CURRENT_USER\Software\OpenShell\OpenShell]
"LastUpdateTime"=dword:1625fe86

[HKEY_CURRENT_USER\Software\OpenShell\OpenShell\Settings]

[HKEY_CURRENT_USER\Software\OpenShell\StartMenu]
"ShowedStyle2"=dword:00000001
"CSettingsDlg"=hex:f0,01,00,00,cb,00,00,00,00,00,00,00,00,00,00,00,36,0d,00,00,\
  01,00,00,00,00,00,00,00
"LastProgramsTime"=hex(b):26,ad,7b,0c,2b,40,db,01
"LastAppsTime"=hex(b):de,3f,74,00,23,1b,db,01
"OldItems"=hex:
"ItemRanks"=hex:48,53,4c,43,01,00,00,00,00,00,00,00,20,f1,44,0d,0a,00,00,00,fb,\
  3c,db,01,1d,65,20,0f,01,00,00,00,83,1b,db,01,7e,e1,00,2c,02,00,00,00,da,36,\
  db,01,7f,4f,ef,2f,01,00,00,00,26,27,db,01,68,8b,93,3c,03,00,00,00,2a,27,db,\
  01,11,3d,d3,42,01,00,00,00,02,1c,db,01,02,46,bd,73,01,00,00,00,04,32,db,01,\
  6b,3e,c4,7e,08,00,00,00,72,3f,db,01,e6,b4,85,80,01,00,00,00,82,2c,db,01,cb,\
  e5,5a,8e,0a,00,00,00,6c,3f,db,01,f9,1b,8e,a7,01,00,00,00,df,32,db,01,d1,ca,\
  b0,cc,08,00,00,00,f5,3c,db,01,ab,36,77,db,02,00,00,00,d7,44,db,01,8b,1c,d7,\
  e2,01,00,00,00,67,21,db,01,7f,f0,0e,f1,02,00,00,00,05,20,db,01,39,19,70,fc,\
  01,00,00,00,d0,20,db,01

[HKEY_CURRENT_USER\Software\OpenShell\StartMenu\MRU]
"0"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\Ps1 To Exe\\Ps1 To Exe.lnk"
"1"="::{26EE0668-A00A-44D7-9371-BEB064C98683}\\0\\::{7B81BE6A-CE2B-4676-A29E-EB907A5126C5}"
"2"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\Mozilla Firefox.lnk"
"3"="C:\\Users\\mzimme2\\AppData\\Roaming\\Microsoft\\Windows\\Start Menu\\Programs\\System Tools\\Command Prompt.lnk"
"4"="::{26EE0668-A00A-44D7-9371-BEB064C98683}\\0\\::{025A5937-A6BE-4686-A844-36FE4BEC8B6D}"
"5"="C:\\Windows\\System32\\cmd.exe"
"6"="::{26EE0668-A00A-44D7-9371-BEB064C98683}\\0\\::{9C73F5E5-7AE7-4E32-A8E8-8D23B85255BF}"
"7"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\Administrative Tools\\Registry Editor.lnk"
"8"="C:\\Windows\\regedit.exe"
"9"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\Visual BCD\\Visual BCD.lnk"
"10"="APP:Microsoft.Windows.Photos_8wekyb3d8bbwe!App"
"11"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\VIP Access.lnk"
"12"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\NetSetMan\\NetSetMan.lnk"

[HKEY_CURRENT_USER\Software\OpenShell\StartMenu\Order]
"DFE05455"=hex:e5,3f,5e,ab,7a,c1,26,57

[HKEY_CURRENT_USER\Software\OpenShell\StartMenu\Settings]
"Version"=dword:040400bf
"SkipMetro"=dword:00000001
"StartScreenShortcut"=dword:00000000



**### HKLM**
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell]

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\ClassicExplorer]
"NewLine"=dword:00000000
"ShowedToolbar"=dword:00000001

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\OpenShell]
"LastUpdateTime"=dword:1626011d

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\OpenShell\Settings]

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu]
"CSettingsDlg"=hex:30,ec,a6,e6,be,01,00,00,70,db,9e,e6,be,01,00,00,30,00,31,00,\
  20,00,28,00,31,00,29,00
"LastProgramsTime"=hex(b):c3,9f,97,c3,0a,45,db,01
"ShowedStyle2"=dword:00000001

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu\Settings]
"AllProgramsMetro"=dword:00000000
"BoldSettings"=dword:00000000
"CheckWinUpdates"=dword:00000000
"DelayIcons"=dword:00000000
"EnableStartButton"=dword:00000000
"HighlightNew"=dword:00000000
"MenuItems7"=hex(7):49,00,74,00,65,00,6d,00,31,00,2e,00,43,00,6f,00,6d,00,6d,\
  00,61,00,6e,00,64,00,3d,00,75,00,73,00,65,00,72,00,5f,00,76,00,69,00,64,00,\
  65,00,6f,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,2e,00,53,00,65,00,74,\
  00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,00,5f,00,44,00,\
  49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,00,32,\
  00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,64,00,6f,00,77,00,\
  6e,00,6c,00,6f,00,61,00,64,00,73,00,00,00,49,00,74,00,65,00,6d,00,32,00,2e,\
  00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,\
  4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,\
  00,65,00,6d,00,33,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,\
  68,00,6f,00,6d,00,65,00,67,00,72,00,6f,00,75,00,70,00,00,00,49,00,74,00,65,\
  00,6d,00,33,00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,\
  49,00,54,00,45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,\
  00,00,00,49,00,74,00,65,00,6d,00,34,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,\
  6e,00,64,00,3d,00,73,00,65,00,70,00,61,00,72,00,61,00,74,00,6f,00,72,00,00,\
  00,49,00,74,00,65,00,6d,00,35,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,\
  64,00,3d,00,67,00,61,00,6d,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,35,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,\
  41,00,43,00,4b,00,5f,00,52,00,45,00,43,00,45,00,4e,00,54,00,7c,00,4e,00,4f,\
  00,45,00,58,00,50,00,41,00,4e,00,44,00,7c,00,49,00,54,00,45,00,4d,00,5f,00,\
  44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,\
  00,36,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,66,00,61,00,\
  76,00,6f,00,72,00,69,00,74,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,36,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,\
  45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,\
  00,74,00,65,00,6d,00,37,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,\
  3d,00,63,00,6f,00,6d,00,70,00,75,00,74,00,65,00,72,00,00,00,49,00,74,00,65,\
  00,6d,00,37,00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,\
  4e,00,4f,00,45,00,58,00,50,00,41,00,4e,00,44,00,00,00,49,00,74,00,65,00,6d,\
  00,38,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,6e,00,65,00,\
  74,00,77,00,6f,00,72,00,6b,00,00,00,49,00,74,00,65,00,6d,00,38,00,2e,00,53,\
  00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,00,\
  5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,\
  00,6d,00,39,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,6e,00,\
  65,00,74,00,77,00,6f,00,72,00,6b,00,5f,00,63,00,6f,00,6e,00,6e,00,65,00,63,\
  00,74,00,69,00,6f,00,6e,00,73,00,00,00,49,00,74,00,65,00,6d,00,39,00,2e,00,\
  53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,\
  00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,\
  65,00,6d,00,31,00,30,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,\
  00,73,00,65,00,70,00,61,00,72,00,61,00,74,00,6f,00,72,00,00,00,49,00,74,00,\
  65,00,6d,00,31,00,31,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,\
  00,63,00,6f,00,6e,00,74,00,72,00,6f,00,6c,00,5f,00,70,00,61,00,6e,00,65,00,\
  6c,00,00,00,49,00,74,00,65,00,6d,00,31,00,31,00,2e,00,53,00,65,00,74,00,74,\
  00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,41,00,43,00,4b,00,5f,00,52,00,\
  45,00,43,00,45,00,4e,00,54,00,00,00,49,00,74,00,65,00,6d,00,31,00,32,00,2e,\
  00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,61,00,64,00,6d,00,69,00,\
  6e,00,00,00,49,00,74,00,65,00,6d,00,31,00,32,00,2e,00,53,00,65,00,74,00,74,\
  00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,41,00,43,00,4b,00,5f,00,52,00,\
  45,00,43,00,45,00,4e,00,54,00,7c,00,49,00,54,00,45,00,4d,00,5f,00,44,00,49,\
  00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,00,31,00,\
  33,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,64,00,65,00,76,\
  00,69,00,63,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,33,00,2e,00,\
  53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,4e,00,4f,00,45,00,58,\
  00,50,00,41,00,4e,00,44,00,00,00,49,00,74,00,65,00,6d,00,31,00,34,00,2e,00,\
  43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,72,00,75,00,6e,00,00,00,49,\
  00,74,00,65,00,6d,00,31,00,35,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,\
  64,00,3d,00,61,00,70,00,70,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,35,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,\
  45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,00,\
  00
"Item1.Command=user_videos"="None"
"SearchAutoComplete"=dword:00000000
"SearchCategories"=dword:00000000
"SearchContents"=dword:00000000
"SearchPrograms"=dword:00000000
"SearchTrack"=dword:00000000
"ShiftClick"="ClassicMenu"
"ShiftWin"="ClassicMenu"
"SkipMetro"=dword:00000001
"StartScreenShortcut"=dword:00000000
"Version"=dword:040400bf

Let me know if you need more information.

**Reply by ge0rdi** [reply]
The HKLM path where start menu settings should be stored is `HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu`.
It should be without `\Settings` at the end.
It is described in `Open-Shell`'s help.

**Comment by MPJ77** [comment]
This is the new registry key I tried.

Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell]

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\ClassicExplorer]
"NewLine"=dword:00000000
"ShowedToolbar"=dword:00000001

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\OpenShell]
"Version"=dword:040400bf
"Path"="C:\\Program Files\\Open-Shell\\"
"DefaultLanguage"=""
"WinVersion"=dword:0a00585d
"LastUpdateTime"=dword:1626011d

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\OpenShell\Settings]

[HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu]
"CSettingsDlg"=hex:30,ec,a6,e6,be,01,00,00,70,db,9e,e6,be,01,00,00,30,00,31,00,\
  20,00,28,00,31,00,29,00
"LastProgramsTime"=hex(b):c3,9f,97,c3,0a,45,db,01
"ShowedStyle2"=dword:00000001
"AllProgramsMetro"=dword:00000000
"BoldSettings"=dword:00000000
"CheckWinUpdates"=dword:00000000
"DelayIcons"=dword:00000000
"EnableStartButton"=dword:00000000
"HighlightNew"=dword:00000000
"MenuItems7"=hex(7):49,00,74,00,65,00,6d,00,31,00,2e,00,43,00,6f,00,6d,00,6d,\
  00,61,00,6e,00,64,00,3d,00,75,00,73,00,65,00,72,00,5f,00,76,00,69,00,64,00,\
  65,00,6f,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,2e,00,53,00,65,00,74,\
  00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,00,5f,00,44,00,\
  49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,00,32,\
  00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,64,00,6f,00,77,00,\
  6e,00,6c,00,6f,00,61,00,64,00,73,00,00,00,49,00,74,00,65,00,6d,00,32,00,2e,\
  00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,\
  4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,\
  00,65,00,6d,00,33,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,\
  68,00,6f,00,6d,00,65,00,67,00,72,00,6f,00,75,00,70,00,00,00,49,00,74,00,65,\
  00,6d,00,33,00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,\
  49,00,54,00,45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,\
  00,00,00,49,00,74,00,65,00,6d,00,34,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,\
  6e,00,64,00,3d,00,73,00,65,00,70,00,61,00,72,00,61,00,74,00,6f,00,72,00,00,\
  00,49,00,74,00,65,00,6d,00,35,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,\
  64,00,3d,00,67,00,61,00,6d,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,35,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,\
  41,00,43,00,4b,00,5f,00,52,00,45,00,43,00,45,00,4e,00,54,00,7c,00,4e,00,4f,\
  00,45,00,58,00,50,00,41,00,4e,00,44,00,7c,00,49,00,54,00,45,00,4d,00,5f,00,\
  44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,\
  00,36,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,66,00,61,00,\
  76,00,6f,00,72,00,69,00,74,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,36,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,\
  45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,\
  00,74,00,65,00,6d,00,37,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,\
  3d,00,63,00,6f,00,6d,00,70,00,75,00,74,00,65,00,72,00,00,00,49,00,74,00,65,\
  00,6d,00,37,00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,\
  4e,00,4f,00,45,00,58,00,50,00,41,00,4e,00,44,00,00,00,49,00,74,00,65,00,6d,\
  00,38,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,6e,00,65,00,\
  74,00,77,00,6f,00,72,00,6b,00,00,00,49,00,74,00,65,00,6d,00,38,00,2e,00,53,\
  00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,00,\
  5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,\
  00,6d,00,39,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,6e,00,\
  65,00,74,00,77,00,6f,00,72,00,6b,00,5f,00,63,00,6f,00,6e,00,6e,00,65,00,63,\
  00,74,00,69,00,6f,00,6e,00,73,00,00,00,49,00,74,00,65,00,6d,00,39,00,2e,00,\
  53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,45,00,4d,\
  00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,\
  65,00,6d,00,31,00,30,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,\
  00,73,00,65,00,70,00,61,00,72,00,61,00,74,00,6f,00,72,00,00,00,49,00,74,00,\
  65,00,6d,00,31,00,31,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,\
  00,63,00,6f,00,6e,00,74,00,72,00,6f,00,6c,00,5f,00,70,00,61,00,6e,00,65,00,\
  6c,00,00,00,49,00,74,00,65,00,6d,00,31,00,31,00,2e,00,53,00,65,00,74,00,74,\
  00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,41,00,43,00,4b,00,5f,00,52,00,\
  45,00,43,00,45,00,4e,00,54,00,00,00,49,00,74,00,65,00,6d,00,31,00,32,00,2e,\
  00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,61,00,64,00,6d,00,69,00,\
  6e,00,00,00,49,00,74,00,65,00,6d,00,31,00,32,00,2e,00,53,00,65,00,74,00,74,\
  00,69,00,6e,00,67,00,73,00,3d,00,54,00,52,00,41,00,43,00,4b,00,5f,00,52,00,\
  45,00,43,00,45,00,4e,00,54,00,7c,00,49,00,54,00,45,00,4d,00,5f,00,44,00,49,\
  00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,49,00,74,00,65,00,6d,00,31,00,\
  33,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,64,00,65,00,76,\
  00,69,00,63,00,65,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,33,00,2e,00,\
  53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,4e,00,4f,00,45,00,58,\
  00,50,00,41,00,4e,00,44,00,00,00,49,00,74,00,65,00,6d,00,31,00,34,00,2e,00,\
  43,00,6f,00,6d,00,6d,00,61,00,6e,00,64,00,3d,00,72,00,75,00,6e,00,00,00,49,\
  00,74,00,65,00,6d,00,31,00,35,00,2e,00,43,00,6f,00,6d,00,6d,00,61,00,6e,00,\
  64,00,3d,00,61,00,70,00,70,00,73,00,00,00,49,00,74,00,65,00,6d,00,31,00,35,\
  00,2e,00,53,00,65,00,74,00,74,00,69,00,6e,00,67,00,73,00,3d,00,49,00,54,00,\
  45,00,4d,00,5f,00,44,00,49,00,53,00,41,00,42,00,4c,00,45,00,44,00,00,00,00,\
  00
"Item1.Command=user_videos"="None"
"SearchAutoComplete"=dword:00000000
"SearchCategories"=dword:00000000
"SearchContents"=dword:00000000
"SearchPrograms"=dword:00000000
"SearchTrack"=dword:00000000
"ShiftClick"="ClassicMenu"
"ShiftWin"="ClassicMenu"
"SkipMetro"=dword:00000001
"StartScreenShortcut"=dword:00000000
"Version"=dword:040400bf


**This is the start screen I get**
![image](https://github.com/user-attachments/assets/e5275af1-c34b-46be-84a7-df5d841288a0)



### **This is the start screen I want**
![image](https://github.com/user-attachments/assets/2e758d15-510f-4a94-8c52-06460bfb2734)

**Reply by ge0rdi** [reply]
So the only difference I can see is that you have _frequently used programs_ enabled in first screen shot.
To disable it you can set `RecentPrograms` value to `None`.
